modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/components/filters/Filter.tsx

```diff
@@ -0,0 +1,135 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayLoadingIndicator from '@clayui/loading-indicator';
+import React, {ReactElement, useContext, useEffect, useState} from 'react';
+
+// @ts-ignore
```

Comment:
Love to see TS grow! :muscle:

Reply:
Yes :-). However I needed to use `ts-ignore` and `any` more than I would have liked, but since the only way to avoid it would be migrating everything I guess we have to live with that... :shrug: